WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/RefundConfirmationViewModel.swift

```diff
@@ -279,3 +287,44 @@ private extension RefundConfirmationViewModel {
         }
     }
 }
+
+private extension WCPayCardBrand {
+    /// A displayable brand name and last 4 digits for a card.
+    /// Names taken from [Stripe's card branding in the API docs](https://stripe.com/docs/api/cards/object#card_object-brand):
+    /// American Express, Diners Club, Discover, JCB, MasterCard, UnionPay, Visa, or Unknown.
+    func localizedCardDetails(last4: String) -> String {
+        return String(format: localizedCardDetailsFormatString(), last4)
+    }
+
+    func localizedCardDetailsFormatString() -> String {
+        switch self {
+        case .amex:
+            return NSLocalizedString("•••• %1$@ (American Express)",
+                                     comment: "American Express card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .diners:
+            return NSLocalizedString("•••• %1$@ (Diners Club)",
+                                     comment: "Diners Club card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .discover:
+            return NSLocalizedString("•••• %1$@ (Discover)",
+                                     comment: "Discover card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .jcb:
+            return NSLocalizedString("•••• %1$@ (JCB)",
+                                     comment: "JCB card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .mastercard:
+            return NSLocalizedString("•••• %1$@ (MasterCard)",
+                                     comment: "MasterCard card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .unionpay:
+            return NSLocalizedString("•••• %1$@ (UnionPay)",
+                                     comment: "UnionPay card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .visa:
+            return NSLocalizedString("•••• %1$@ (Visa)",
+                                     comment: "Visa card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .unknown:
+            return NSLocalizedString("•••• %1$@ (Unknown provider)",
```

Comment:
Not sure if this is an actual possibility, or just a necessary fallback, but maybe we can just skip the `(Unknown provider)` here? I can imagine that being more confusing than helpful